interface/main/backup.php

```diff
@@ -163,7 +163,7 @@ function gzopen($filename, $mode, $use_include_path = 0)
    echo "  <td>" . xl('Create Eventlog Backup') . "</td>\n";
    echo " </tr>\n";
    echo " <tr>\n";
-   echo "  <td></td><td class='text'><b>" . xl('Note')."</b>&nbsp;" . xl('Please refer to').'&nbsp;README-Log-Backup.txt&nbsp;'.xl('file in the Documentation directory to learn how to automate the process of creating log backups') . "</td>\n";
+   echo "  <td></td><td class='text'><strong>" . xl('Note').":</strong>&nbsp;" . xl('Please refer to').'&nbsp;README-Log-Backup.txt&nbsp;'.xl('file in the Documentation directory to learn how to automate the process of creating log backups') . "</td>\n";
```

Comment:
Huuhh will this do the trick i doubt.... 🤔 . Testing though...

Reply:
I dont know why git highlights this line but IMO this line doesn't fix the title display issue

Reply:
Not sure if my predictions are correct 😆